lumicks/pylake/channel.py

```diff
@@ -62,6 +62,11 @@ def timestamps(self):
         """Absolute timestamps (since epoch) which correspond to the channel data"""
         return self._src.timestamps
 
+    @property
+    def time(self):
+        """Relative time (in seconds) that corresponds to the channel data"""
+        return (self._src.timestamps-self._src.timestamps[0])/1e9
```

Comment:
Very tiny nitpick, but a space between the variables and the operator would be nice.

`(self._src.timestamps - self._src.timestamps[0])/1e9`